README.adoc

```diff
@@ -90,14 +90,31 @@ Build the example located in the _upstream-1_ folder:
     git commit -a -m"Pages update"
     git push origin +gh-pages
     
-=== Creating References to Anchors
+=== Creating Anchors and xrefs
 
+==== Dynamic Anchor and Dynamic xref
 An anchor (for example: [[Testing the App]]) cannot appear more than once in an *.adoc* file.
 As Modular Doc files are regularly reused in the same *.adoc* file, this results in a build error.
 See link:http://asciidoctor.org/docs/user-manual/#include-multiple[Including Multiple Anchors] to negate this issue.
 
 To best understand the information above, see the example located in the repo.
 
+==== Dynamic Anchors and Hard-Coded xref
```

Comment:
Ahhhh-  it's there because I made this PR when _I didn't know how to properly work with upstream and forks_ - it's basically a headache I have caused (i'm guessing you are seeing conflicts as I made the change in (https://github.com/feedhenry-raincatcher/raincatcher-docs/pull/20) - my apologies .